src/components/visualizations/chart/highcharts/getOptionalStackingConfiguration.ts

```diff
@@ -48,17 +54,23 @@ function handleStackMeasuresToPercent(stackMeasuresToPercent: boolean, seriesIte
         : seriesItem;
 }
 
-function handleDualAxis(isDualAxis: boolean, seriesItem: ISeriesItem): ISeriesItem {
-    if (!isDualAxis) {
+function handleDualAxis(chartOptions: IChartOptions, seriesItem: ISeriesItem): ISeriesItem {
+    const { yAxes, type } = chartOptions;
+    const isDualAxis = yAxes.length === 2;
+
+    if (!isDualAxis && !isComboChart(type)) {
         return seriesItem;
     }
 
     const { stacking, yAxis } = seriesItem;
-    const isFirstAxis = yAxis === 0;
+    const seriesChartType = seriesItem.type || type;
+    const defaultStackingValue = isComboChart(type) ? null : NORMAL_STACK;
+    const customStackingValue =
```

Comment:
I would say the logic for computing the right stackingValue could be extracted from this method :) just use here something like `getCurrentStackingValue(...), getVisualizationStackingValue, getStackingValue,...` or something similar :)